### PR TITLE
Pubstwo 1365

### PR DESCRIPTION
**Pull request by ayan-usgs**

### CODE REVIEW

File: pubs_ui/manager/static/js/hb_templates/bibliodata.hbs [hunk 1]
```diff
@@ -136,13 +135,20 @@
 			<input id="publisher-location-input" type="text" class="form-control"/>
 		</div>
 	</div>
-	<div class="form-group" id="publication-year-div">
-		<label class="control-label" for="isbn-input">Publication Year</label>
+    <div class="form-group" id="publication-year-div">
```
Comment (mbucknell) [inline pubs_ui/manager/static/js/hb_templates/bibliodata.hbs]: The templates should be formatted using tabs. In general we use tabs for indentation in everything except python (which of course requires spaces).

File: pubs_ui/manager/static/js/hb_templates/bibliodata.hbs [hunk 2]
```diff
@@ -136,13 +135,20 @@
...
+        <label class="control-label" for="isbn-input">Publication Year</label>
```
Comment (mbucknell) [inline pubs_ui/manager/static/js/hb_templates/bibliodata.hbs]: The for should be "publication-year-input"

File: pubs_ui/manager/static/js/hb_templates/bibliodata.hbs [hunk 3]
```diff
@@ -136,13 +135,20 @@
...
-		<div>
-			<input id="publication-year-input" type="text" class="form-control"/>
-		</div>
-	</div>
+        <div>
+            <input id="publication-year-input" type="text" class="form-control"/>
+        </div>
+    </div>
+    <div class="form-group" id="no-year-div">
+        <label class="control-label" for="isbn-input">No Publication Year</label>
```
Comment (mbucknell) [inline pubs_ui/manager/static/js/hb_templates/bibliodata.hbs]: The for should be "no-year-input"

File: tests/js/manager/models/PublicationModelSpec.js [hunk 4]
```diff
@@ -146,6 +146,23 @@ define([
 				expect(server.requests[0].requestBody).toEqual('{"id" : 1234}');
 			});
 
+			it('Expects that a call to release with noYear succeeds.', function() {
```
Comment (mbucknell) [inline tests/js/manager/models/PublicationModelSpec.js]: Is this testing any new code.  It's up to the server to tell us if we can release a publication and there where no changes in PublicationModel for this ticket. Consider removing this test.

File: tests/js/manager/models/PublicationModelSpec.js [hunk 5]
```diff
@@ -216,6 +233,23 @@ define([
 				expect(server.requests[0].requestBody).toEqual('{"id" : 1234}');
 			});
 
+			it('Expects that a call to publish with noYear succeeds.', function() {
+				server.respond([200, {"Content-Type" : 'application/json'},
```
Comment (mbucknell) [inline tests/js/manager/models/PublicationModelSpec.js]: Same comment as above.

**Comment by mbucknell** [comment]
Some comments to address but otherwise 👍

File: pubs_ui/manager/static/js/hb_templates/bibliodata.hbs (hunk 1 above)
Comment (ayan-usgs) [inline pubs_ui/manager/static/js/hb_templates/bibliodata.hbs]: Okay.

**Pull request merged**